scripts/bootloader/keygen.py

```diff
@@ -23,10 +23,11 @@ def generate_legal_key():
     while True:
         key = ec.generate_private_key(ec.SECP256R1())
         public_bytes = key.public_key().public_bytes(
-                encoding=serialization.Encoding.DER,
-                format=serialization.PublicFormat.SubjectPublicKeyInfo,
-                )
-        digest = sha256(public_bytes).digest()[:16]
+                encoding=serialization.Encoding.X962,
+                format=serialization.PublicFormat.UncompressedPoint,
+        )
+
+        digest = sha256(public_bytes[1:]).digest()[:16]
```

Comment:
Just incase anyone wonder why we take off 1 byte here it's the encoding output which gives 0x4 for uncompressed points. It wasn't possible to just get the Raw EC points out of the library in the same way as the python implementation.

Reply:
How about adding a comment on that?

Reply:
added